pkg/apiserver/webhook/configmap_webhook.go

```diff
@@ -106,24 +110,28 @@ func (v *ConfigMapValidator) validateUpdate(req admission.Request) admission.Res
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
-	oldCloudResourcePrefix, err := v.getCloudResourcePrefix(oldConfigMap)
+	oldControllerConfig, err := v.getControllerConfig(oldConfigMap)
 	if err != nil {
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
-	if newCloudResourcePrefix == oldCloudResourcePrefix {
-		return admission.Allowed("")
-	}
-	if v.NpControllerInterface.IsCloudResourceCreated() {
-		return admission.Denied(fmt.Sprintf("cloud resource is already created with CloudResourcePrefix %s. "+
-			"Please delete the resource and try or use the same CloudResourcePrefix",
-			oldCloudResourcePrefix))
+	if newControllerConfig.CloudResourcePrefix != oldControllerConfig.CloudResourcePrefix {
+		if v.NpControllerInterface.IsCloudResourceCreated() {
+			return admission.Denied(fmt.Sprintf("cloud resource is already created with CloudResourcePrefix %s. "+
+				"Please delete the resource and try or use the same CloudResourcePrefix", oldControllerConfig.CloudResourcePrefix))
+		}
+		if len(newControllerConfig.CloudResourcePrefix) > 0 {
+			if !config.ValidateName(newControllerConfig.CloudResourcePrefix) {
+				return admission.Denied(fmt.Sprintf("invalid CloudResourcePrefix %s, "+
+					"First and last characters should be alphanumeric and '-' characters are allowed only in the middle.",
+					newControllerConfig.CloudResourcePrefix))
+			}
+		}
 	}
-	if len(newCloudResourcePrefix) > 0 {
-		if !config.ValidateName(newCloudResourcePrefix) {
-			return admission.Denied(fmt.Sprintf("invalid CloudResourcePrefix %s, "+
-				"First and last characters should be alphanumeric and '-' characters are allowed only in the middle.",
-				newCloudResourcePrefix))
+	if newControllerConfig.CloudSyncInterval != oldControllerConfig.CloudSyncInterval {
+		if newControllerConfig.CloudSyncInterval < 60 {
```

Comment:
nit: may be have a constant defined for the minimum value.

Reply:
Solved.